src/Setup/WixBA/WixBA.cs

```diff
@@ -108,12 +142,22 @@ public static void PlanLayout()
                 WixBA.Plan(WixBA.Model.Command.Action);
             }
         }
+        /// <summary>
+        /// Helper to attach a debugger to the cr instance when debugging
+        /// </summary>
+        [Conditional("DEBUG")]
+        private static void AttachDebuger()
+        {
+            System.Diagnostics.Debugger.Launch();
+            return;
+        }
 
         /// <summary>
         /// Thread entry point for WiX Toolset UX.
         /// </summary>
         protected override void Run()
         {
+            // AttachDebuger();
```

Comment:
I find this to be a useful tool now that the bundle creates a cr.  It certainly can be removed 'when we're done'.